src/configureWorkspace/configure.ts

```diff
@@ -385,6 +386,7 @@ async function configureCore(context: IActionContext, options: ConfigureApiOptio
         os = await quickPickOS();
     }
     properties.configureOs = os;
+    properties.orchestration = 'single' as DockerOrchestration;
```

Comment:
Is `as DockerOrchestration` needed here and below?

Reply:
Strictly speaking no, but I can't do `properties.orchestration: DockerOrchestration = 'single'` and I didn't want it as typeless as a simple string...